### PR TITLE
formatting: cleared all outputs

### PR DESCRIPTION
**Pull request by keylbill**
# Existing Sample Changes
## Description

Formatting/maintenance: cleared all outputs for all notebook samples owned by AITCE

## External Dependencies

n/a

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used
- [X] JupyterLab

### CODE REVIEW

**Comment by keylbill** [comment]
@louie-tsai @raoberman

**Review by louie-tsai — COMMENTED** [review]
please focus on the notebooks with outputs. For those notebooks without outputs, please remove the changes from the PR. thanks

File: AI-and-Analytics/End-to-end-Workloads/Census/census_modin.ipynb [hunk 1]
```diff
@@ -306,9 +306,9 @@
    "name": "python",
```
Comment (louie-tsai) [inline AI-and-Analytics/End-to-end-Workloads/Census/census_modin.ipynb:306]: same here. possible to remove this changes for census if it is only for version? 
you could checkout the file before you create a commit.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/tutorial_optimize_TensorFlow_pretrained_model.ipynb [hunk 2]
```diff
@@ -439,20 +439,13 @@
...
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
    "display_name": "intel-tensorflow",
    "language": "python",
-   "name": "intel-tensorflow"
+   "name": "python3"
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/tutorial_optimize_TensorFlow_pretrained_model.ipynb:448]: possible to remove the changes which are only for version?

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/.ipynb_checkpoints/ResNet50_Inference-checkpoint.ipynb [hunk 3]
```diff
@@ -110,20 +110,13 @@
...
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
    "display_name": "Python [conda env:user_tensorflow] *",
    "language": "python",
-   "name": "conda-env-user_tensorflow-py"
+   "name": "python3"
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/.ipynb_checkpoints/ResNet50_Inference-checkpoint.ipynb:119]: same here. possible to reverse the change?

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/ResNet50_Inference.ipynb [hunk 4]
```diff
@@ -110,20 +110,13 @@
...
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
    "display_name": "Python [conda env:user_tensorflow] *",
    "language": "python",
-   "name": "conda-env-user_tensorflow-py"
+   "name": "python3"
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/ResNet50_Inference.ipynb:119]: same here. possible to reverse the change?

**Review by louie-tsai — COMMENTED** [review]
still have some unnecessary changes for python version difference.

File: AI-and-Analytics/Getting-Started-Samples/IntelPython_XGBoost_GettingStarted/IntelPython_XGBoost_GettingStarted.ipynb [hunk 5]
```diff
@@ -160,43 +144,28 @@
...
+   ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.7 (Intel® oneAPI)",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "c009-intel_distribution_of_python_3_oneapi-beta05-python"
+   "name": "python3"
```
Comment (raoberman) [inline AI-and-Analytics/Getting-Started-Samples/IntelPython_XGBoost_GettingStarted/IntelPython_XGBoost_GettingStarted.ipynb:168]: same comment as louie - maybe OK but may not pass CI

File: AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult/Intel_Extension_for_SKLearn_Performance_SVC_Adult.ipynb [hunk 6]
```diff
@@ -381,9 +275,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.7 (Intel® oneAPI)",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "c009-intel_distribution_of_python_3_oneapi-beta05-python"
+   "name": "python3"
```
Comment (raoberman) [inline AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult/Intel_Extension_for_SKLearn_Performance_SVC_Adult.ipynb:280]: same comment here

File: AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult/Intel_Extension_for_SKLearn_Performance_SVC_Adult.ipynb [hunk 7]
```diff
@@ -12,7 +12,7 @@
   },
```
Comment (raoberman) [inline AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult/Intel_Extension_for_SKLearn_Performance_SVC_Adult.ipynb:12]: For this file, can you also remove the line:  "In cases when unsupported parameters are used, the package fallbacks into original Scikit-learn. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/intel/scikit-learn-intelex/issues)." in the readme? Thanks

**Comment by raoberman** [comment]
> still have some unnecessary changes for python version difference.

Same comment for the rest of the code samples. Be careful when modifying the kernelspec, not sure if that's necessary for this. See also my comment on modifying one of the READMEs

**Comment by keylbill** [comment]
@louie-tsai @raoberman ,
reverted all unnecessary changes in kernelspec for all files. Please check if it's satisfactory.

**Review by louie-tsai — APPROVED** [review]
looks good to me. thanks

**Comment by louie-tsai** [comment]
@jimmytwei 
possible to also merge this PR?

**Review by jimmytwei — APPROVED** [review]

**Pull request merged**